feature-libs/user/profile/root/user-profile-root.module.ts

```diff
@@ -2,12 +2,13 @@ import { NgModule } from '@angular/core';
 import {
   provideDefaultConfigFactory,
   UserProfileFacadeTransitionalToken,
+  UserRegisterFacadeTransitionalToken,
 } from '@spartacus/core';
 import {
   USER_PROFILE_CORE_FEATURE,
   USER_PROFILE_FEATURE,
 } from './feature-name';
-import { UserProfileFacade } from './facade';
+import { UserProfileFacade, UserRegisterFacade } from './facade';
```

Comment:
It's better to import straight from the file, or from the barrel (index.ts) file

Reply:
👍